.github/labeler.yml

```diff
@@ -13,6 +13,7 @@ Documentation:
 
 Bash:
   - ./*.sh
+  - apps/**/*.sh
```

Comment:
```suggestion
  - apps/**/*.sh
  - conf/**/*.sh
  - deps/**/*.sh
  - modules/**/*.sh
```

Reply:
Your right, looks that way. Thanks @Kitzunu

Reply:
@Kitzunu actually I think we might still need the 
`./*.sh`
otherwise it wont pickup the acore.sh and install.sh in the root. Do you agree?

Reply:
I tested this on a seperate repo and ./ seems to be as broken as the any: and all: sadly

Reply:
Gotcha, sounds good. I have added it again.